c/src/concept/concept.rs

```diff
@@ -56,7 +56,7 @@ pub extern "C" fn value_new_string(string: *const c_char) -> *mut Concept {
 /// Creates a new ``Value`` object of the specified datetime value.
 #[no_mangle]
 pub extern "C" fn value_new_date_time_from_millis(millis: i64) -> *mut Concept {
-    release(Concept::Value(Value::DateTime(NaiveDateTime::from_timestamp_millis(millis).unwrap())))
+    release(Concept::Value(Value::DateTime(DateTime::from_timestamp_millis(millis).unwrap().naive_utc())))
```

Comment:
Conversions between naive datetime and UNIX timestamp are deprecated and must now be performed via `DateTime<Utc>`